doc/ref_cert/lfn/chapters/chapter03.md

```diff
@@ -17,28 +17,47 @@
   * [3.8.2 Resiliency Measurements](#3.8.2)
 * [3.9 NFVI Test Cases](#3.9)
 
+
 <a name="3.1"></a>
 ## 3.1 Introduction
 
-> Scope of this chapter is to have a list of test cases needed (a detailed table of sort)
+The objective of this chapter is to describe the requirements on NFVI test cases as derived from the reference model and architecture for the LFN-based compliance program. This set of requirements eventually determines the scope of the compliance program and the corresponding list of test cases included in the complaince program.  In particular, this chapter extends the generic list of NFVI test case requirements which is provided in Chapter 8.2.4 of the reference model.
+
 
 <a name="3.2"></a>
-## 3.2 Assumptions
+## 3.2 Generic Requirements
+
+All test cases must fullfil the generic requirements listed in Chapter 8.2.4 of the reference model.
+
+In addition, for test cases targeting the NFVI compliance program, the following requirements must be met:
+
+| Reference         | Description                                                                                                              |
+|-------------------|--------------------------------------------------------------------------------------------------------------------------|
+| x                 | All NFVI test cases *must* be automated, that is, implemented in an open source  test tool.                              |
```

Comment:
```All NFVI test cases *must* be automated, that is, implemented in an open source  test tool.```
I think this can be mentioned as two separate requirements.
 - All NFVI test cases *must* be automated
 - All NFVI test cases *must* be implemented using open source ~~test~~ tools

Reply:
Done.  Updated.  Resolved/Closed.